src/pipecat/services/xtts.py

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import aiohttp
+
+from typing import AsyncGenerator
+
+from pipecat.frames.frames import AudioRawFrame, ErrorFrame, Frame
+from pipecat.services.ai_services import TTSService
+
+from loguru import logger
+
+import requests
+
+import numpy as np
+import resampy
+
+#####
+## The server below can connect to XTTS through a local running docker
+##
+## Docker command: $ docker run --gpus=all -e COQUI_TOS_AGREED=1 --rm -p 8000:80 ghcr.io/coqui-ai/xtts-streaming-server:latest-cuda121
+## 
+## You can find more information on the official repo: https://github.com/coqui-ai/xtts-streaming-server
+####
+
+SERVER_URL = 'http://localhost:8000'
```

Comment:
Probably better to add this as a constructor `base_url` argument. This way you can pass the server you want, see for example `DeepgramTTSService`.

Reply:
ok, I will update and re-submit.